src/components/Counter.js

```diff
@@ -20,10 +27,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={/*() => this.props.increment() */this.incrementIfOdd}>
```

Comment:
Another way to do this is by passing in the function directly because the anonymous function isn't necessary here. You can do that like this, `onClick={this.props.increment}`.